
Use anonymous shm for ldap cache, though it's still broken.  

--- httpd-2.0.47/modules/experimental/util_ldap_cache.c.ldapshm
+++ httpd-2.0.47/modules/experimental/util_ldap_cache.c
@@ -295,14 +295,7 @@
 #if APR_HAS_SHARED_MEMORY
     apr_status_t result;
 
-    result = apr_shm_create(&util_ldap_shm, reqsize, MODLDAP_SHMEM_CACHE, pool);
-    if (result == EEXIST) {
-        /*
-         * The cache could have already been created (i.e. we may be a child process).  See
-         * if we can attach to the existing shared memory
-         */
-        result = apr_shm_attach(&util_ldap_shm, MODLDAP_SHMEM_CACHE, pool);
-    } 
+    result = apr_shm_create(&util_ldap_shm, reqsize, NULL, pool);
     if (result != APR_SUCCESS) {
         return result;
     }
